packages/desktop/package.json

```diff
@@ -68,10 +68,11 @@
   },
   "homepage": "https://github.com/TryQuiet",
   "@comment version": "To build new version for specific platform, just replace platform in version tag to one of following linux, mac, windows",
-  "version": "0.8.3-alpha.0",
+  "version": "0.8.3-alpha.18",
```

Comment:
I'm not sure if we want to bump it this way

Reply:
u should refresh the changes, i reverted that change